.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,30 @@
+---
+name: ":question: Question"
+about: Ask a question or request general help support.
+title: ":question: [Question]: "
+labels: question
+assignees: tlvu
```

Comment:
For support I guess @MatProv could also help?

Reply:
Yes, absolutely